### PR TITLE
Bug 2032 - No Pending task being created for Response

### PR DESCRIPTION
**Pull request by nitish-beehyv**
created/show/close pending task for citizen and did dev testing
#2032

### CODE REVIEW

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/submissions/src/pages/employee/SubmissionsCreate.js [hunk 1]
```diff
@@ -510,15 +510,16 @@ const SubmissionsCreate = ({ path }) => {
...
-    const assignes = !isAssignedRole ? [userInfo?.uuid] || [] : [];
+    const assignes = assignees ? assignees : !isAssignedRole ? [userInfo?.uuid] || [] : [];
     await submissionService.customApiService(Urls.application.pendingTask, {
       pendingTask: {
         name,
         entityType,
         referenceId: `MANUAL_${refId}`,
         status,
-        assignedTo: assignes?.map((uuid) => ({ uuid })),
+        assignedTo: assignees ? assignes : assignes?.map((uuid) => ({ uuid })),
```
Comment (vaibhavct) [inline frontend/micro-ui/web/micro-ui-internals/packages/modules/submissions/src/pages/employee/SubmissionsCreate.js:522]: Remove the condition from here. Declare const assignees  as const finalAssigness and map those here.
finalAssignees should always have assignees in it

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/submissions/src/pages/employee/SubmissionsCreate.js [hunk 2]
```diff
@@ -822,12 +822,21 @@ const SubmissionsCreate = ({ path }) => {
...
           setShowSuccessModal(true);
+
           createPendingTask({ name: t("MAKE_PAYMENT_SUBMISSION"), status: "MAKE_PAYMENT_SUBMISSION", isCompleted: true });
+
+          if (applicationDetails?.additionalDetails?.isResponseRequired === true) {
+            createPendingTask({
+              name: t("RESPOND_TO_PRODUCTION_DOCUMENTS"),
+              status: "RESPOND_TO_PRODUCTION_DOCUMENTS",
+              stateSla: orderDetails?.orderDetails?.dates?.submissionDeadlineDate,
+              assignees: applicationDetails?.additionalDetails?.respondingParty?.map((item) => ({ uuid: item?.uuid[0] })),
```
Comment (vaibhavct) [inline frontend/micro-ui/web/micro-ui-internals/packages/modules/submissions/src/pages/employee/SubmissionsCreate.js:837]: Don't map first only first uuid. respondingParty will have list of uuid, map it here

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/submissions/src/pages/employee/SubmissionsCreate.js (hunk 1 above)
Comment (nitish-beehyv) [reply frontend/micro-ui/web/micro-ui-internals/packages/modules/submissions/src/pages/employee/SubmissionsCreate.js]: done

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/submissions/src/pages/employee/SubmissionsCreate.js (hunk 2 above)
Comment (nitish-beehyv) [reply frontend/micro-ui/web/micro-ui-internals/packages/modules/submissions/src/pages/employee/SubmissionsCreate.js]: done

**Pull request closed without merging**